src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -35,7 +35,7 @@ public class TryConvertTool : ITryConvertTool
                 throw new ArgumentNullException(nameof(tryConvertOptionsAccessor));
             }
 
-            Path = Environment.ExpandEnvironmentVariables(tryConvertOptionsAccessor.Value.TryConvertPath);
+            Path = tryConvertOptionsAccessor.Value.TryConvertPath;
```

Comment:
I'd leave the `Environment.ExpandEnvironmentVariables`. It won't be needed 99% of the time, but we're pulling the path from a user-modifiable config file so who knows what will end up there.